documentation/copy notes.md

```diff
@@ -0,0 +1,41 @@
+# Copy notes
+
+This document is a work in progress and might be subject to change. 
+**_Last update: August 23, 2024_**
+
+## Forms
+### Accessibility
+-	**Checkbox and radio button questions**: either introduce the options with a generic label and present full sentences for selection, or use a label that is a question and provide complete answers.
+
+### Point of view
+-	Typically, we use the first person to describe answers in a form.
+    * E.g.: [   ] I am willing to work occasional overtime. 
+    * And not: [   ] ~~You are willing to work occasional overtime~~.
+
+## Punctuation
+-	Generally, we don’t use **quotation marks** for answers in forms.
+    * Exception: Indigenous identity – In forms providing the possibility to indicate identification with an Indigenous community, quotation marks are intentionally included to emphasize the significance.
+- Avoid using **slashes** when possible.
+    * For example, say "City or town" and not "~~City/town~~"
+  
+
+## Impératif vs. infinitif
+Generally:
+-	Use infinitif for headings
+-	Use impératif for descriptions
+-	Use impératif for buttons that entail a concrete action
+
+However:
+-	Use impératif for headings that include a possessive pronoun/adjective (often found on pages that request concrete info or actions from the users)
+-	Use infinitif for buttons that are more generic. (For example that lead to more information (e.g. Learn more, Expand all, View results, Browse jobs))
+
+See for more info on the distinction between impératif and inifinitif the website of the [Office quebecois de la langue française](https://vitrinelinguistique.oqlf.gouv.qc.ca/24208/la-grammaire/le-verbe/modes/imperatif/distinction-entre-limperatif-et-linfinitif).
+## Capitalization
+[…]
```

Comment:
```suggestion
Coming soon.
```

Reply:
Not sure if this is a placeholder, but better to say what it is than [...] which is markdown.

Reply:
good point, removed it altogether and will re-add when I have something concrete.